index.src.html

```diff
@@ -1652,11 +1656,20 @@
                 |svc|.{{SvcOutputMetadata/temporalLayerId}}.
             4. Assign |svc| to
                 |chunkMetadata|.{{EncodedVideoChunkMetadata/svc}}.
-        8. If |encoderConfig|.{{VideoEncoderConfig/alpha}} is set to `"keep"`:
+        8. If |encoderConfig|.{{VideoEncoderConfig/scalabilityMode}}
+            describes multiple [=spatial layers=]:
+            1. Let |svc| be a new {{SvcOutputMetadata}} instance.
```

Comment:
This will override everything produced by the step 8 for SVC modes that have both temporal and spatial layers. for example: L2T2

Reply:
I think I have fixed this. PTAL.